### PR TITLE
Benny's branch

### PR DESCRIPTION
**Pull request by benny-n**
-created basic UI menus logic
-added cursor toggle option

### CODE REVIEW

File: The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (orenkoren) [inline The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml:1]: don't commit this file

Comment (orenkoren) [reply The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml:1]: or any file in this PR that has ./idea in it

File: The Middle Ages/.idea/.idea.The Middle Ages/riderModule.iml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (orenkoren) [inline The Middle Ages/.idea/.idea.The Middle Ages/riderModule.iml:1]: don't commit files related to your IDE

File: The Middle Ages/Assets/Game/Scripts/Database/Abilities.cs [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 namespace MiddleAges.Database
 {
-    public static class Abilities
+    public static class Abilities
```
Comment (orenkoren) [inline The Middle Ages/Assets/Game/Scripts/Database/Abilities.cs:6]: ?

File: The Middle Ages/Assets/Game/Scripts/Menus/Menu.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+public abstract class Menu : MonoBehaviour
+{
+    protected static bool IsClosed { get; set; }
+    // Start is called before the first frame update
```
Comment (orenkoren) [inline The Middle Ages/Assets/Game/Scripts/Menus/Menu.cs:11]: remove this

File: The Middle Ages/The Middle Ages.sln [hunk 5]
```diff
@@ -1,24 +1,18 @@
 ﻿
```
Comment (orenkoren) [inline The Middle Ages/The Middle Ages.sln:1]: dont commit this file

File: The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (benny-n) [reply The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml:1]: added to gitignore

File: The Middle Ages/.idea/.idea.The Middle Ages/riderModule.iml (hunk 2 above)
Comment (benny-n) [reply The Middle Ages/.idea/.idea.The Middle Ages/riderModule.iml:1]: added to gitignore

File: The Middle Ages/Assets/Game/Scripts/Database/Abilities.cs (hunk 3 above)
Comment (benny-n) [reply The Middle Ages/Assets/Game/Scripts/Database/Abilities.cs:6]: probably ide bug cause I dont have this space

File: The Middle Ages/Assets/Game/Scripts/Menus/Menu.cs (hunk 4 above)
Comment (benny-n) [reply The Middle Ages/Assets/Game/Scripts/Menus/Menu.cs]: done

File: The Middle Ages/The Middle Ages.sln (hunk 5 above)
Comment (benny-n) [reply The Middle Ages/The Middle Ages.sln:1]: added to gitignore

File: The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml (hunk 1 above)
Comment (orenkoren) [reply The Middle Ages/.idea/.idea.The Middle Ages/.idea/codeStyles/codeStyleConfig.xml:1]: it's still here tho

**Pull request closed without merging**